src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1465,16 +1465,36 @@ public function isMethodSafe()
      */
     public function getContent($asResource = false)
     {
-        if (PHP_VERSION_ID < 50600 && (false === $this->content || (true === $asResource && null !== $this->content))) {
+        $currentContentIsResource = is_resource($this->content);
+        if (PHP_VERSION_ID < 50600 && (false === $this->content || (true === $asResource && null !== $this->content && !$currentContentIsResource))) {
```

Comment:
Is the check for `$currentContentIsResource` really necessary? I mean is PHP not capable to handle two streams reading from `php://temp` (I would have guessed that the temporary filenames would differ when using it multiple times)?